src/classes/VOL_CTRL_VolunteersReportHours.cls

```diff
@@ -264,6 +274,13 @@ global virtual with sharing class VOL_CTRL_VolunteersReportHours {
         		listHours[0].Start_Date__c = vhours.Start_Date__c;
         		listHours[0].End_Date__c = vhours.End_Date__c;
         		listHours[0].Hours_Worked__c = vhours.Hours_Worked__c;
+        	
+        		// update the additional fields, if any
+        		for (Schema.FieldSetMember f : SObjectType.Volunteer_Hours__c.FieldSets.VolunteersReportHoursFS.getFields()) {
+        			String fieldName = f.getFieldPath();
+		        	listHours[0].put(fieldName, vhours.get(fieldName));		
+		        }	
+        		   		
```

Comment:
this copy field logic also needs to be included when creating new hours, not just the update case.  Also, in the update case, we should only copy over the new value if it is not null, so we don't overwrite existing info that might be useful.